index.bs

```diff
@@ -2202,6 +2202,45 @@ During the above process, the user agent SHOULD show some UI to the user to guid
 authorizing an authenticator.
 </div>
 
+#### Registration API Exceptions
+
+[=[WRPS]=] can encounter the following {{DOMException|DOMExceptions}} from a call to {{CredentialsContainer/create()|navigator.credentials.create()}}. Some errors can have multiple reasons for why they happened, requiring the [=[WRPS]=] to infer the actual reason based on their use of WebAuthn:
+
+<dl dfn-type="argument" dfn-for="Registration API Exceptions">
+    :   <dfn>AbortError</dfn>
```

Comment:
These `<dfn>`s are making existing references ambiguous, see Bikeshed output below.

Do we need to refer to these locations in the document, or could we use references instead of definitions here?

(This also applies to the authentication errors section)

```suggestion
<dl>
    :   {{AbortError}}
```

```
LINK ERROR: Ambiguous for-less link for 'NotAllowedError', please see <https://speced.github.io/bikeshed/#ambi-for> for instructions:
Local references:
spec:webauthn-3; type:argument; for:Registration API Exceptions; text:NotAllowedError
spec:webauthn-3; type:argument; for:Authentication API Exceptions; text:NotAllowedError
for-less references:
spec:webidl; type:exception; for:/; text:NotAllowedError
{{NotAllowedError}}
LINK ERROR: Ambiguous for-less link for 'TypeError', please see <https://speced.github.io/bikeshed/#ambi-for> for instructions:
Local references:
spec:webauthn-3; type:argument; for:Registration API Exceptions; text:TypeError
for-less references:
spec:webidl; type:exception; for:/; text:TypeError
{{TypeError}}
LINK ERROR: Ambiguous for-less link for 'SecurityError', please see <https://speced.github.io/bikeshed/#ambi-for> for instructions:
Local references:
spec:webauthn-3; type:argument; for:Registration API Exceptions; text:SecurityError
spec:webauthn-3; type:argument; for:Authentication API Exceptions; text:SecurityError
for-less references:
spec:webidl; type:exception; for:/; text:SecurityError
{{SecurityError}}
LINK ERROR: Ambiguous for-less link for 'NotSupportedError', please see <https://speced.github.io/bikeshed/#ambi-for> for instructions:
Local references:
spec:webauthn-3; type:argument; for:Registration API Exceptions; text:NotSupportedError
for-less references:
spec:webidl; type:exception; for:/; text:NotSupportedError
{{NotSupportedError}}
LINK ERROR: Ambiguous for-less link for 'InvalidStateError', please see <https://speced.github.io/bikeshed/#ambi-for> for instructions:
Local references:
spec:webauthn-3; type:argument; for:Registration API Exceptions; text:InvalidStateError
for-less references:
spec:webidl; type:exception; for:/; text:InvalidStateError
{{InvalidStateError}}
LINK ERROR: Multiple possible 'idl' local refs for 'userVerification'.
Randomly chose one of them; other instances might get a different random choice.
{{userVerification}}
LINK ERROR: Multiple possible 'idl' local refs for 'excludeCredentials'.
Randomly chose one of them; other instances might get a different random choice.
{{excludeCredentials}}
LINK ERROR: Ambiguous for-less link for 'UnknownError', please see <https://speced.github.io/bikeshed/#ambi-for> for instructions:
Local references:
spec:webauthn-3; type:argument; for:Registration API Exceptions; text:UnknownError
spec:webauthn-3; type:argument; for:Authentication API Exceptions; text:UnknownError
for-less references:
spec:webidl; type:exception; for:/; text:UnknownError
{{UnknownError}}
LINK ERROR: Ambiguous for-less link for 'ConstraintError', please see <https://speced.github.io/bikeshed/#ambi-for> for instructions:
Local references:
spec:webauthn-3; type:argument; for:Registration API Exceptions; text:ConstraintError
for-less references:
spec:webidl; type:exception; for:/; text:ConstraintError
{{ConstraintError}}
```